packages/jaeger-ui/src/components/TracePage/index.test.js

```diff
@@ -369,10 +347,18 @@ describe('mapDispatchToProps()', () => {
 });
 
 describe('mapStateToProps()', () => {
-  it('maps state to props correctly', () => {
-    const id = 'abc';
-    const trace = {};
-    const state = {
+  const id = 'abc';
```

Comment:
clearer variable names and values in such a large file could help. since this is a traceID (I believe), the variable name and value could be traceID. It interferes with object shorthand below but is clearer if there are errors.